src/collisions/COLLISIONS_alloc_and_free.F

```diff
@@ -191,10 +191,18 @@ subroutine local_alloc_and_free(COLL,COLL_grp)
            YAMBO_ALLOC(COLL(J)%v_c,(COLL(J)%N))
            COLL(J)%v_c=cZERO
            mem_size(4)=mem_size(4)+size(COLL(J)%v_c)
+           if(SP==DP.and.COLLISIONS_load_SP) then
+             YAMBO_ALLOC(COLL(J)%v_c_SP,(COLL(J)%N))
+             COLL(J)%v_c_SP=cZERO
+             mem_size(4)=mem_size(4)+size(COLL(J)%v_c_SP)
+           endif
          endif
        endif
        if (l_clean.and.(l_v1.or.l_all)) then
          YAMBO_FREE(COLL(J)%v_c)
+         if(SP/=DP.and.allocated(COLL(J)%v_c_SP)) then
```

Comment:
- The allocation is done if (SP==DP.and.COLLISIONS_load_SP) which the allocation is done with the opposite condition. Is this correct? 
- YAMBO_FREE(x) already does a check "if(allocated(x))", no need to put it again.

Reply:
I just fixed this, and running test again to be sure everything is fine.
I had a small fail in yambo_rt